tema27112019/src/main/java/ex4_Produs/Magazin.java

```diff
@@ -0,0 +1,16 @@
+package ex4_Produs;
+
+import java.util.Map;
+
+public class Magazin {
```

Comment:
Uitate in diagrama atasata , iti lipsesc niste metode din magazin  https://drive.google.com/file/d/1Iw24E4r-tBRPzimzDRZN7zWEn5VGYx6O/view?usp=sharing